motoman_sia20d_descartes/include/motoman_sia20d_descartes/motoman_sia20d_robot_model.h

```diff
@@ -34,6 +34,13 @@ namespace motoman_sia20d_descartes
 
     virtual bool getAllIK(const Eigen::Affine3d &pose, std::vector<std::vector<double> > &joint_poses) const;
 
+    virtual descartes_core::RobotModelPtr clone() const
+    {
+      descartes_core::RobotModelPtr ptr (new MotomanSia20dRobotModel);
+      ptr->initialize("robot_description", descartes_moveit::MoveitStateAdapter::group_name_, world_frame_, tool_frame_);
+      return ptr;
+    }
```

Comment:
I believe this was originally added to work with a test version of Descartes (involving multiple threads) and isn't needed anymore.